File: dataset/human/java/file_2553.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		String  s  = sc.next();
		String  t  = sc.next();
		char[]  new_s = s.toCharArray();
		char[]  new_t = t.toCharArray();
		Arrays.sort(new_s);
		Arrays.sort(new_t);
		String _s = new String(new_s);
		String _t = new StringBuffer( new String(new_t) ).reverse().toString();
		if(_s.compareTo(_t)<0) System.out.println("Yes");
		else	                   System.out.println("No");
	}

}

File: dataset/human/c/file_2553.c
#include <stdio.h>

int main(void){
	int i;
	char s[3];
	 for(i=0;i<3;i++){
		scanf("%c",&s[i]);
        }
	if(s[0]=='a'||s[1]=='a'||s[2]=='a'){
		if(s[0]=='b'||s[1]=='b'||s[2]=='b'){
			if(s[0]=='c'||s[1]=='c'||s[2]=='c') {
				printf("Yes\n");
			}else  printf("No\n");
		}else printf("No\n");
	}else printf("No\n");
	return 0;
}
